src/index.html

```diff
@@ -1,21 +1,12 @@
 <!DOCTYPE html>
-<html>
```

Comment:
Is this a best practice somewhere? I saw something about Google recommending it. I just haven't seen this around much and am wondering if it's a good idea.